src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub struct Mod {
 impl Mod {
     pub fn new(value: i32, modulus: i32) -> Mod {
         let remainder = value % modulus;
-        let value = match remainder > 0 {
+        let value = match remainder >= 0 {
```

Comment:
I was afraid of you asking for tests here. I tried to.

The thing is I was not able to write a failing test as our constructor would make this pass:
```rust
assert!(Mod::new(5,5) == Mod::new(0,5))
# and both mod would have value 5 due to the "bug"
```

Writing a failing test here would mean testing the `value` value, *i.e* checking internal state of the structure (which is private), hence producing an error:
```rust
assert!(Mod::new(5,5).value == 0)
# error: field `value` of struct `crypto_tools::Mod` is private
```

I do not think exposing `value` as public a good idea yet (I mean, we do not need it elsewhere than in the tests for now).

---

In the end, my argument is that this small change does not break any public interface, and our structure is behaving as we expect it to behave regarding to the operations we tested so far.
Maybe later, we will need to control internal values (because of Mathematic reasons), and only then we will write tests for these cases.